### PR TITLE
NOV-239933: set webview to pagelements

### PR DESCRIPTION
**Pull request by hesteban-tuenti**
When using toolium in a mobile context like the one in we have in Novum, where we have the native context and the webview context with a bunch of different webviews and we have defined elements of class PageElements that should be located within the webview context, the current implementation won't work because the pageelements are located in the native context by definition.

To cope with it so far we have been defining a parent element with the webview attribute set to true, and passing it in the initializer of the PageElements element. 

Mostly this does the trick with the caveat to have to define extra elements in the pageobject but recently we found an scenario where also iframes are involved and this doesn't work [NOV-239995](https://jira.tid.es/browse/NOV-239995)

### CODE REVIEW

File: toolium/pageobjects/common_object.py [hunk 1]
```diff
@@ -62,3 +64,56 @@ def utils(self):
...
+                context, window_handle = self.webview_context_selection_callback(*self.webview_csc_args)
+            elif self.parent and self.parent.webview_context_selection_callback:
+                context, window_handle = self.parent.webview_context_selection_callback(*self.parent.webview_csc_args)
+            else:
+                app_web_context = "{}_{}".format(CommonObject.webview_context_prefix,
+                                                 self.driver.capabilities['appPackage'])
+                if app_web_context in self.driver.contexts:
+                    context = app_web_context
+                    if self.driver.context != context:
+                        self.driver.switch_to.context(context)
```
Comment (rgonalo) [inline toolium/pageobjects/common_object.py:84]: Para reducir complejidad y evitar repetir estas dos líneas, se puede hacer un método "switch_to_new_context(context)" que haga la comprobación y el switch. Se usaría 3 veces en este método y otras 2 en el de ios.

**Comment by hesteban-tuenti** [comment]
there is an error in the tests with python 3.9. I don't think it's related with the changes but I don't have permissions to re-launch the job action .

**Review by angel-ram — APPROVED** [review]
LGTM

File: toolium/pageobjects/common_object.py [hunk 2]
```diff
@@ -62,3 +64,57 @@ def utils(self):
...
+                context_id = self.webview_context_selection_callback(*self.webview_csc_args)
+            else:
+                contexts = self.driver.execute_script('mobile: getContexts')
+                context_id = next(
+                    (item['id'] for item in reversed(contexts) if
+                     'bundleId' in item and item['bundleId'] == self.driver.capabilities['bundleId']),
+                    None)
+            if context_id:
+                if self.driver.context != context_id:
+                    self.driver.switch_to.context(context_id)
```
Comment (rgonalo) [inline toolium/pageobjects/common_object.py:116]: use self._switch_to_new_context(self, context)

Comment (hesteban-tuenti) [reply toolium/pageobjects/common_object.py]: arggh, fixed !

**Review by rgonalo — APPROVED** [review]

**Review by angel-ram — APPROVED** [review]

**Pull request merged**